{"name": "test_access_points_connectivity", "status": "passed", "description": "Test case to verify Access Points Connectivity", "attachments": [{"name": "Access Point Connectivity Failed", "source": "10a41ae1-e461-47c7-9aa0-b638d44fc628-attachment.attach"}, {"name": "Manager status is Active", "source": "de8bd527-ca9a-4425-a4a7-3e3bf7fe330c-attachment.attach"}, {"name": "log", "source": "86a20388-d593-4f96-960e-7bd141d68cc2-attachment.txt", "type": "text/plain"}], "start": 1627615284964, "stop": 1627615284965, "uuid": "544b75fa-f068-469a-a551-595f6a224a8e", "historyId": "9c9c1794599fbd8dfa1d00f937bef4d8", "testCaseId": "b1de51dd692b456a6a233ecea89ec452", "fullName": "test_connectivity.TestResources#test_access_points_connectivity", "labels": [{"name": "tag", "value": "test_access_points_connectivity"}, {"name": "tag", "value": "test_resources"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup_test_run')"}, {"name": "tag", "value": "sanity_55"}, {"name": "suite", "value": "test_connectivity"}, {"name": "subSuite", "value": "TestResources"}, {"name": "host", "value": "testing-97-dczmk"}, {"name": "thread", "value": "6-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_connectivity"}], "links": [{"type": "test_case", "name": "test_access_points_connectivity"}, {"type": "test_case", "name": "Test Resources"}]}